### PR TITLE
Rna seq

### LINKED ISSUE #171: BUG: [put title here]
**Describe the bug**
MultiQC sample name parsing doesn't handle assembly-sample names. Was inconsistent as well.

### PR DESCRIPTION
**Pull request by siebrenf**
**What problem is the PR solving / What's new?**
fixed #171 
Also fixed several minor issues

**What did change**
added command line config to multiqc.

**Does this need additional test(s)?**
No

### CODE REVIEW

File: rules/assembly_hub.smk [hunk 1]
```diff
@@ -63,5 +64,5 @@ rule bam_bigwig:
         deeptools_limit=1
     shell:
         """
-        bamCoverage --bam {input} --outFileName {output} --numberOfProcessors {threads} {params} --verbose >> {log} 2>&1
+        bamCoverage --bam {input.bam} --outFileName {output} --numberOfProcessors {threads} {params} --verbose >> {log} 2>&1
         """
```
Comment (Maarten-vd-Sande) [inline rules/assembly_hub.smk]: .bai file not used if im correct?

File: workflows/atac_seq/Snakefile [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
-
 # complement the configuration
 configfile: "config.yaml"
 
 
-for schema in ['general', 'download', 'alignment_general', 'peakcalling']:
-    validate(config, schema=f"../../schemas/config/{schema}.schema.yaml")
-
 # do onstart/onexit things
+sample_schemas = ['sample', 'assembly', 'strandedness', 'condition']
```
Comment (Maarten-vd-Sande) [inline workflows/atac_seq/Snakefile]: I can't find where this is used

Comment (Maarten-vd-Sande) [reply workflows/atac_seq/Snakefile]: this = strandedness

File: rules/assembly_hub.smk (hunk 1 above)
Comment (siebrenf) [reply rules/assembly_hub.smk]: not in the command line, but the script fails if it is not present.

**Comment by Maarten-vd-Sande** [comment]
Why is Jenkins here twice as check? branch and pr-merge?

**Pull request merged**